File: 0001 Code Chef/0006 Christmas Greetings/0005 kotlin/ChristmasGreetings.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    var day:Int? = stringInput?.toIntOrNull();
    if(day != null){
       if(day == 25){
        println("YES");
       }else{
        println("NO");
       } 
    } 
}

File: 0001 Code Chef/0006 Christmas Greetings/implementation/0003 c#/ChristmasGreetings.cs
using System;

class ChristmasGreetings{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int day = Convert.ToInt32(stringInput);
        if(day == 25){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}